testing/jormungandr-automation/resources/explorer/graphql/schema.graphql

```diff
@@ -44,7 +44,10 @@ type BlockConnection {
   pageInfo: PageInfo!
 
   """A list of edges."""
-  edges: [BlockEdge]
+  edges: [BlockEdge!]!
+
+  """A list of nodes."""
+  nodes: [Block!]!
   totalCount: Int!
```

Comment:
Why making BlockEdge not nullable?

Reply:
This was autogenerated after running the generate schema script. We updated the graphql-async lib last week, im guessing it could be related to this? Nothing else has changed.